README.md

```diff
@@ -120,3 +127,11 @@ sh scripts/evaluate_solver.sh \
         data/ARC-V1-Feb2018/ARC-Challenge/ARC-Challenge-Test.jsonl \
         my_awesome_model/
     ``` 
+    
+    
+## Running the BiLSTM max-out Question to Choices Max Attention
+To run the model, you need to have the data and embeddings downloaded.
+
+```bash
```

Comment:
Can you expand this to include the commands to train and evaluate on the trained model ? Also use a `models` or similar folder name instead of `temp`.

Reply:
I will use trained_models folder. I still do not have the trained models of the SOA model. There is a minor change of the model, due to the downgrade of allennlp and I have to do hyper-parameter tuning to get a similar result to the best model.